db/migrate/20250511040228_create_favorites.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFavorites < ActiveRecord::Migration[8.0]
+  def change
+    create_table :favorites do |t|
+      t.timestamps
+      t.references :user, null: false, foreign_key: true
```

Comment:
`["user_id", "post_id"]`のユニークインデックスを作っているので、ここではindex: falseして重複するインデックスが無いようにしたい
```suggestion
      t.references :user, null: false, foreign_key: true, index: false
```